frontend/src/views/producer/dashboard/edition-form/Editor.jsx

```diff
@@ -237,6 +240,17 @@ class Editor extends React.Component {
       };
     }
   };
+
+  updateCoverImageName = imageurl => {
+    this.internalData.eventDraft.coverImage = imageurl;
+    this.uploadProgress(false);
+    this.internalData.eventDraft.uploadFailed = imageurl ? false : true;
+  };
+
+  uploadProgress = value => {
+    this.internalData.eventDraft.uploadInProgress = value;
```

Comment:
`internalData.uploadInProgress`. This upload in progress or upload failed is not part of eventDraft